packages/repository/src/relations/has-many/has-many.decorator.ts

```diff
@@ -17,7 +21,7 @@ import {HasManyDefinition, RelationType} from '../relation.types';
  */
 export function hasMany<T extends Entity>(
   targetResolver: EntityResolver<T>,
-  definition?: Partial<HasManyDefinition>,
+  definition?: Partial<HasManyDefinition | HasManyThroughDefinition>,
 ) {
   return function(decoratedTarget: object, key: string) {
     const meta: HasManyDefinition = Object.assign(
```

Comment:
This is not correct for "has-many-through" case, is it?

See the discussion in for https://github.com/strongloop/loopback-next/pull/2359#discussion_r317011708 for more context.